### PR TITLE
CMR-5699

### PR DESCRIPTION
**Pull request by littleidiot40**
https://bugs.earthdata.nasa.gov/browse/CMR-5699
Provider sitemaps should link to all CMR collection landing pages

Currently, if a collection has a DOI then the provider sitemap will not list the CMR landing page. This means Google Dataset Search will not index the CMR collection landing page in these instances.

We should do the following for all CMR collections with a DOI in the metadata,

Link to the DOI in https://cmr.earthdata.nasa.gov/search/site/collections/directory/<provider>/gov.nasa.eosdis
Add the cmr collection landing page link in https://cmr.earthdata.nasa.gov/search/site/collections/directory/<provider>/gov.nasa.eosdis/sitemap.xml

### CODE REVIEW

File: oracle-lib/README.md [hunk 1]
```diff
@@ -25,8 +27,11 @@ project.
 
   In each case, be sure you get the version matching the versions specified
   in the `project.clj`.
```
Comment (CayvonH) [inline oracle-lib/README.md]: The markdown here doesn't seem to be formatted properly:
https://github.com/nasa/Common-Metadata-Repository/blob/b8afa03ddc5d31a729e89ef030e3ac826505ffb4/oracle-lib/README.md

File: oracle-lib/README.md [hunk 2]
```diff
@@ -25,8 +27,11 @@ project.
...
-2. Put the Jars in `oracle-lib/support`
-3. Finally, perform the actual installation: `cmr install oracle-libs`.
+2. `cd oracle-lib`
+3. `mkdir support`
+4. Put the Jars in `oracle-lib/support`
+5. From `oracle-lib`, run `./support/install_oracle_jars.sh` which will
```
Comment (CayvonH) [inline oracle-lib/README.md]: Where is this script?

File: oracle-lib/README.md (hunk 1 above)
Comment (littleidiot40) [reply oracle-lib/README.md]: Odd. It's resolving correctly locally in my viewer. Cheers.

Comment (littleidiot40) [reply oracle-lib/README.md]: Fixed.

File: oracle-lib/README.md (hunk 2 above)
Comment (littleidiot40) [reply oracle-lib/README.md]: The oracle README was not current with the project. Updated.

File: oracle-lib/README.md [hunk 3]
```diff
@@ -16,17 +16,19 @@ project.
...
 
-  In each case, be sure you get the version matching the versions specified
-  in the `project.clj`.
-2. Put the Jars in `oracle-lib/support`
-3. Finally, perform the actual installation: `cmr install oracle-libs`.
+In each case, be sure you get the version matching the versions specified in the `project.clj`.
+1. `cd oracle-lib`
+2. `mkdir support`
+3. Put the Jars in `oracle-lib/support`
+4. In the project root directory, run `cmr install oracle-libs` which will
```
Comment (CayvonH) [inline oracle-lib/README.md]: I believe this should be ./bin/cmr if running from the root directory.

File: oracle-lib/README.md [hunk 4]
```diff
@@ -16,17 +16,19 @@ project.
 
 ### Manually installing the Oracle JDBC Jars
 
-1. Download oracle Jars from [Oracle's website.](https://www.oracle.com/technetwork/apps-tech/jdbc-112010-090769.html) You need to download the following:
+Download oracle Jars from Oracle's website. You can usually google for each of the jar files with a search term such as "oracle ucp jar" to find the location of each jar file. You need to download the following:
```
Comment (mschmele) [inline oracle-lib/README.md]: I had originally hoped to avoid having to tell people to 'just google it' - does that link not work anymore?

**Pull request merged**